### PR TITLE
0084-feel-for-loops

### PR DESCRIPTION
**Pull request by StrayAlien**

### CODE REVIEW

File: TestCases/compliance-level-3/0084-feel-for-loops/0084-feel-for-loops.dmn [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <variable name="decision_002"/>
+        <literalExpression>
+            <text>for i in [1,2,3], j in [4,5] return i + j</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_004" id="_decision_004">
+        <variable name="decision_004"/>
+        <literalExpression>
+            <text>for i in (2..4] return i</text>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0084-feel-for-loops/0084-feel-for-loops.dmn]: There are still test which in my perspective are invalid from the DMNv1.2 grammar as noted in https://github.com/dmn-tck/tck/pull/204#pullrequestreview-194537530

**Comment by StrayAlien** [comment]
Thanks @tarilabs. Those tests have been removed.  I have added two extra tests - one for an empty list and another for a numeric range with the same upper/lower endpoints like `1..1`.

**Comment by opatrascoiu** [comment]
looks good to me.
